src/pages/homeCollection/kitAssembly.js

```diff
@@ -358,15 +358,13 @@ const closeAlert = (status = "warn", duration = 5000) => {
   }
 };
 
-const dropdownTrigger = (sitekeyName) => {
-  let dropdownSiteBtn = document.getElementById('dropdownSites');
-  let dropdownMenuButton = document.getElementById('dropdownMenuButtonSites');
-  let tempSiteName = dropdownSiteBtn.innerHTML = sitekeyName;
+const dropdownTrigger = () => {
```

Comment:
It is called in line 111 (`dropdownTrigger('Select Kit Type');`), and the string argument isn't needed.